### DISCUSSION TITLE
Would like to convert a CSV while retaining leading zeros

### DISCUSSION THREAD

**Discussion by David-Globe-Electric** [dfinke/ImportExcel#1313] [General]
I love your PS module!

I am having trouble finding examples of how to take an existing CSV file and converting it to Excel using your tool.
An added bonus is retaining leading zeros, which AFAIK, is possible using the -NoNumberConversion switch.

I watched the youtube playlist and rifled through the examples. Maybe I missed it :(

Thanks in advance!

**Comment by dfinke** [comment]
@David-Globe-Electric thank you! May not have that example here is a way:

```powershell
Import-Csv .\salesData.csv | Export-Excel .\salesData.xlsx -NoNumberConversion -Show
```

**Reply by David-Globe-Electric** [reply]
Thanks! However, I gave that a swirl and got an error saying that -NoNumberConversion was missing an argument.

![image](https://user-images.githubusercontent.com/118382574/202304990-13fbe5cb-3d6d-45d2-851b-6fc66a0ee691.png)

**Comment by dfinke** [comment]
Sorry, you need to specify the name of the property to do the ` -NoNumberConversion` on

**Comment by David-Globe-Electric** [comment]
Thank you so much! This is so useful! 

For future readers, I simply specified the column names separated by commas:

Import-Csv .\FiletoConvert.csv | Export-Excel .\ConvertedFile.xlsx -NoNumberConversion columnx, columny -Show

Columnx and y are simply the headers of the columns in the CSV file.
